week-05-202402/Headers/Exercise4/ControladoraOvni.h

```diff
@@ -0,0 +1,154 @@
+#pragma once
+#include "Alfa.h"
+#include "Beta.h"
+#include "Gamma.h"
+
+class Controladora
+{
+private:
+	vector<Alfa*> alfas;
+	vector<Beta*> betas;
+	vector<Gamma*> gammas;
```

Comment:
It might be a good idea to have a single vector<Ovni*> taking advantage of inheritance